HeliPort/Supporting files/itl80211_security+Description.swift

```diff
@@ -44,3 +44,11 @@ extension itl80211_security: CustomStringConvertible {
         }
     }
 }
+
+extension itl80211_security: Encodable {
```

Comment:
`Codable` is `Encodable` + `Decodable` so just conforming to this is enough

```suggestion
extension itl80211_security: Codable { }
```

Reply:
Thanks, but seems the patch doesn't remove the following lines?

Reply:
You have to do a manual commit I guess. Hard to do in this Github web ui